wagtailio/services/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from wagtail.admin import edit_handlers
+from wagtail.core import blocks, fields
+from wagtail.core.models import Page
+
+from wagtailio.services.blocks import SectionBlock
+
+
+class ServicesPage(Page):
+    intro = models.TextField(null=True, blank=True)
+    body = fields.StreamField(
+        blocks.StreamBlock(
+            [('section', SectionBlock())],
+            min_num=4,
+            max_num=4,
```

Comment:
question: was it a requirement to have to add 4? seems like it might make for a nicer editorial experience to add as few or many as they like but i'm not familiar with the requirements of the page : )

Reply:
~~That is a fair question. The design I have received had four sections. I assumed that was fixed. But you are right. I will go back and double check that requirement.~~ I just checked my notes and it says "Section block (Struct - 4 exactly)" so I guess I had clarified that before 😅